### PR TITLE
Configure Client Profiles for JM, TH, MT

### PR DESCRIPTION
**Pull request by annaliseirby090**
### Checklist
- [ ] Corresponding issue has been opened
- [ ] New tests added
- [ ] Feature flags added
- [ ] Strings are localized
- [ ] Tested for chat contacts
- [ ] Tested for call contacts

### Other Related Issues

None

### AFTER YOU MERGE

1. Cut a release tag using the Github workflow. Wait for it to complete and notify in the #aselo-deploys Slack channel.
2. Comment on the ticket with the release tag version AND any additional instructions required to configure an environment to test the changes.
3. Only then move the ticket into the QA column in JIRA

You are responsible for ensuring the above steps are completed. If you move a ticket into QA without advising what version to test, the QA team will assume the latest tag has the changes. If it does not, the following confusion is on you! :-P

### CODE REVIEW

File: hrm-form-definitions/form-definitions/jm/v1/profileForms/FlagDurations.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "flag": "abusive profanity",
+    "label": "Abusive - Profanity"
+  },
+  {
+    "flag": "risk level low green",
+    "label": "Risk Level : Low (Green)"
```
Comment (janorivera) [inline hrm-form-definitions/form-definitions/jm/v1/profileForms/FlagDurations.json:8]: These first four will not work I think, flag durations work for existing flags. 
Profile flags are added in [here ](https://github.com/techmatters/hrm/blob/master/hrm-domain/hrm-service/scripts/README.md) using commands. I can help you with this :)

File: hrm-form-definitions/form-definitions/th/v1/profileForms/FlagDurations.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "flag": "blocked",
+    "label": "Long Term Block (100 Years)",
```
Comment (janorivera) [inline hrm-form-definitions/form-definitions/th/v1/profileForms/FlagDurations.json:4]: 100 years ?? my gosh ! XD

**Review by janorivera — CHANGES_REQUESTED** [review]
just the flag durations will need to be changed

File: hrm-form-definitions/form-definitions/th/v1/profileForms/FlagDurations.json (hunk 2 above)
Comment (annaliseirby090) [reply hrm-form-definitions/form-definitions/th/v1/profileForms/FlagDurations.json:4]: Hahaha yes they asked for an effectively permanent block so we offered 100 years!

File: hrm-form-definitions/form-definitions/jm/v1/profileForms/FlagDurations.json (hunk 1 above)
Comment (annaliseirby090) [reply hrm-form-definitions/form-definitions/jm/v1/profileForms/FlagDurations.json:8]: Ahhh thank you! Sorry I forgot that part. Could you provide more specific instructions or hop on a quick call?

File: hrm-form-definitions/form-definitions/jm/v1/profileForms/FlagDurations.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  {
+    "flag": "risk level moderate amber",
+    "label": "Risk Level : Moderate (Amber)"
+  },
```
[the remark this answers is not in the document]
Comment (annaliseirby090) [reply hrm-form-definitions/form-definitions/jm/v1/profileForms/FlagDurations.json:13]: Hi - from @janorivera 's comments below, it looks like I added this in the wrong place. So I'll remove this!

File: hrm-form-definitions/form-definitions/th/v1/profileForms/FlagDurations.json (hunk 2 above)
Comment (mythilytm) [reply hrm-form-definitions/form-definitions/th/v1/profileForms/FlagDurations.json:4]: 😆

File: hrm-form-definitions/form-definitions/jm/v1/profileForms/FlagDurations.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (mythilytm) [reply hrm-form-definitions/form-definitions/jm/v1/profileForms/FlagDurations.json:13]: 👍

[the remark this answers is not in the document]
Comment (annaliseirby090) [reply hrm-form-definitions/form-definitions/jm/v1/profileForms/FlagDurations.json]: Hi @mythilytm @janorivera I just removed the flags without durations! Please take a look and let me know if you approve :D

**Review by mythilytm — APPROVED** [review]

File: hrm-form-definitions/form-definitions/jm/v1/profileForms/FlagDurations.json (hunk 1 above)
Comment (annaliseirby090) [reply hrm-form-definitions/form-definitions/jm/v1/profileForms/FlagDurations.json]: finished during a 1-1 call!

**Review by janorivera — APPROVED** [review]

**Pull request merged**